PYTHON/generate_manifest.py

```diff
@@ -102,7 +102,9 @@ def browse_directories(dir_path: str, cur_type: Optional[str] = None):
             __generated_nodes.append(n_file_name)
         except Exception as e:
             print(
-                "❌ Failed to generate manifest from ",
+                "❌ Failed to generate manifest from ".encode("ascii", "replace").decode(
```

Comment:
Why does this need ascii encoding? Does UTF-8 break things? If it's the emojis breaking we could probably just remove them